docs/copied-from-beats/shared-template-load.asciidoc

```diff
@@ -0,0 +1,254 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/shared-template-load.asciidoc[]
+//// If you want to include conditional content, you also need to
+//// add the following doc attribute definition  before the
+//// include statement so that you have:
+//// :allplatforms:
+//// include::../../libbeat/docs/shared-template-load.asciidoc[]
+//// This content must be embedded underneath a level 3 heading.
+//////////////////////////////////////////////////////////////////////////
+
+
+In Elasticsearch, {elasticsearch}/indices-templates.html[index
+templates] are used to define settings and mappings that determine how fields
+should be analyzed.
+
+The recommended index template file for {beatname_uc} is installed by the
+{beatname_uc} packages. If you accept the default configuration in the
++{beatname_lc}.yml+ config file, {beatname_uc} loads the template automatically
+after successfully connecting to Elasticsearch. If the template already exists,
+it's not overwritten unless you configure {beatname_uc} to do so.
+
+You can disable automatic template loading, or load your own template, by
+configuring template loading options in the {beatname_uc} configuration file.
+
+You can also set options to change the name of the index and index template.
+
+ifndef::only-elasticsearch[]
+NOTE: A connection to Elasticsearch is required to load the index template. If
+the output is Logstash, you must
```

Comment:
I would remove all descriptions for Logstash output as we don't describe it anywhere as possible output option.

Reply:
see the `ifndef::only-elasticsearch[]` line above. `only-elasticsearch` is defined for APM Server so these lines should not show up.